python/lsst/ip/isr/crosstalk.py

```diff
@@ -163,15 +169,31 @@ def fromDict(cls, dictionary):
                                f"found {dictionary['metadata']['OBSTYPE']}")
 
         calib.setMetadata(dictionary['metadata'])
-        calib._detectorName = dictionary['metadata']['DETECTOR']
-        calib._detectorSerial = dictionary['metadata']['DETECTOR_SERIAL']
+        calib._detectorName = dictionary.get('detectorName',
+                                             dictionary['metadata'].get('DETECTOR', None))
+        calib._detectorSerial = dictionary.get('detectorSerial',
+                                               dictionary['metadata'].get('DET_SER', None))
+        calib._instrument = dictionary.get('instrument',
+                                           dictionary['metadata'].get('INSTRUME', None))
 
         calib.hasCrosstalk = dictionary.get('hasCrosstalk',
                                             dictionary['metadata'].get('HAS_CROSSTALK', False))
         if calib.hasCrosstalk:
             calib.nAmp = dictionary.get('nAmp', dictionary['metadata'].get('NAMP', 0))
             calib.crosstalkShape = (calib.nAmp, calib.nAmp)
```

Comment:
I wonder here why we don't have symmetry with the other properties, and just have the dict hold its shape info rather than reconstructing from `nAmps`. I mean, I'm sure it'll always be fine, but, you know...

Reply:
The issue was persisting to FITS, which doesn't support tuples in the metadata.  It instead sets `CROSSTALK_SHAPE` once each for the two entries.

Reply:
Yuk, OK. Marked as resolved.